content/kubeone/master/architecture/requirements/infrastructure_management/_index.en.md

```diff
@@ -144,6 +145,19 @@ The needed `output.tf` file already comes with all our
 [terraform-configs-github]: https://github.com/kubermatic/kubeone/tree/master/examples/terraform
 {{% /notice %}}
 
+## VMware
+
+For the vSphere csi plugin to work correctly machines created in vSphere need to meet the [prerequisites][vsphere-storage].
+This is done automatically by the machine-controller or the provided terraform examples, therefore it is only relevant when machines are created manually.
+
+Make sure the that all VMs have the disk.enableUUID flag set to 1, this can be done using the [govc tool](https://github.com/vmware/govmomi/tree/master/govc) with the following command:
```

Comment:
```suggestion
Make sure that all VMs have the `disk.enableUUID` flag set to 1. This can be done using the [govc tool](https://github.com/vmware/govmomi/tree/master/govc) with the following command:
```

Also, please make sure the URL is handled in the same way as other URLs.